### PR TITLE
feat: ITV-29 add alterations page and remove structural changes page

### PR DESCRIPTION
**Pull request by LBHABEBIC**
# What?

Added new step after the lamington flooring. it replaces structural changes which has been removed. 

# Notes

image

### CODE REVIEW

File: steps/property-inspection/alterations.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+          }),
+        })
+      ),
+      ComponentWrapper.wrapDynamic(
+        new DynamicComponent({
+          key: "alterations-notes",
+          Component: PostVisitActionInput,
+          props: {
+            label: {
+              value: "Add note about alterations made to porperty by tenant.",
```
Comment (mrmartineau) [inline steps/property-inspection/alterations.tsx]: Please fix the spelling
```suggestion
              value: "Add note about alterations made to property by tenant.",
```

File: steps/property-inspection/alterations.tsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  step: {
+    slug: PageSlugs.Alterations,
+    nextSlug: PageSlugs.Garden,
+    submit: (nextSlug?: string): ReturnType<typeof makeSubmit> =>
+      makeSubmit({
+        slug: nextSlug as PageSlugs | undefined,
+        value: "Save and continue",
+      }),
+    componentWrappers: [
+      ComponentWrapper.wrapDynamic(
```
Comment (mrmartineau) [inline steps/property-inspection/alterations.tsx]: It looks like you are missing the two paragraphs before this first question 
image

**Review by mrmartineau — CHANGES_REQUESTED** [review]
It looks like some things are missing, please see my comments

File: steps/property-inspection/alterations.tsx (hunk 1 above)
Comment (LBHABEBIC) [reply steps/property-inspection/alterations.tsx]: @mrmartineau 
These should be fixed now.

File: steps/property-inspection/alterations.tsx [hunk 3]
```diff
@@ -64,6 +66,33 @@ const step: ProcessStepDefinition<ProcessDatabaseSchema, "property"> = {
...
+          key: "paragraph-2",
+          Component: Paragraph,
+          props: {
+            children: (
+              //TODO add correct url
+              <>
+                Information about alterations is in{" "}
+                <Link href={""} target="_blank">
+                  [Your Tenancy conditions: section 4.19]
+                </Link>{" "}
```
Comment (mrmartineau) [inline steps/property-inspection/alterations.tsx]: What should the `href` value for this link be? Do we know it?

Also, Balsamiq adds square brackets around text to show that they are  links. Can you remove them here please?

Comment (mrmartineau) [reply steps/property-inspection/alterations.tsx]:
```suggestion
                <Link href={""} target="_blank">
                  Your Tenancy conditions: section 4.19
                </Link>{" "}
```

**Review by mrmartineau — CHANGES_REQUESTED** [review]

**Review by mrmartineau — APPROVED** [review]
👍

**Review by LBH-wgreeff — APPROVED** [review]
looks good

**Pull request merged**